packages/dm-core-plugins/src/form/widgets/DimensionalScalarWidget.tsx

```diff
@@ -55,11 +55,11 @@ const DimensionalScalarWidget = (props: TWidget) => {
         style={
           widgetConfig?.inline
             ? {
-                border: 'solid #6c6c6c 1px',
+                // borderBottom: 'solid #6c6c6c 1px',
                 borderRadius: '2px 0 0 2px',
                 display: 'flex',
                 alignItems: 'center',
-                background: '#dedede',
+                // background: '#f7f7f7',
```

Comment:
Could we group these colors somewhere?
Many hard coded hex codes...

Reply:
yes, there is some colors in tailwind.config.

Reply:
now uses reusable colors bg-equinor-lightgray